react/features/dial-out/components/DialOutNumbersForm.web.js

```diff
@@ -24,13 +24,6 @@ const DEFAULT_COUNTRY = {
 };
 
 /**
- * The expand icon of the dropdown menu.
- *
- * @type {ReactElement}
- */
-const EXPAND_ICON = <ExpandIcon label = 'expand' />;
```

Comment:
I don't necessarily prefer inlining these. Rather the linter yelled at me after I added line breaks for props and wrapped the jsx in parentheses. I decided I'd rather inline the icon to stop the errors.